chemprop/cli/utils/parsing.py

```diff
@@ -262,16 +263,29 @@ def load_input_feats_and_descs(paths, n_molecules, feat_desc):
 
 
 def make_dataset(
-    data: Sequence[MoleculeDatapoint] | Sequence[ReactionDatapoint], reaction_mode: str
+    data: Sequence[MoleculeDatapoint] | Sequence[ReactionDatapoint], reaction_mode: str, multi_hot_atom_featurizer_mode: str = "DEFAULT",
 ) -> MoleculeDataset | ReactionDataset:
+    
+    match multi_hot_atom_featurizer_mode:
+        case "DEFAULT":
+            atom_featurizer=MultiHotAtomFeaturizer.default()
+        
+        case "V1":
+            atom_featurizer=MultiHotAtomFeaturizer.v1()
+            
+        case "ORGANIC":
+            atom_featurizer=MultiHotAtomFeaturizer.organic()
+     
```

Comment:
do we need to raise an error for an unknow multi_hot_atom_featurizer_mode?

Reply:
There should be a new enum:
```python
class AtomFeatureMode(EnumMapping):
    DEFAULT = auto()
    V1 = auto()
    ORGANIC = auto()
```
so that the unknown case will be handled by that and we can throw a `RuntimeError` if it falls through the `match`-statement